source/operations/server-side-encryption/configure-minio-kes.rst

```diff
@@ -0,0 +1,220 @@
+      :end-before: end-requires-operator-plugin
+
+   See :ref:`deploy-operator-kubernetes` for complete documentation on deploying the MinIO Operator.
+
+.. _minio-sse-vault-prereq-vault:
+
+Ensure KES Access to a Supported KMS Target
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. cond:: linux or macos or windows or container
+
+   This procedure assumes an existing KES installation connected to a supported |KMS| installation accessible, both accessible from the local host.
+   Refer to the installation instructions for your :kes-docs:`supported KMS target <#supported-kms-targets>` to deploy KES and connect it to a KMS solution.
+   
+   .. admonition:: KES Operations Require Unsealed Target
+      :class: important
+   
+      Some supported |KMS| targets allow you to seal or unseal the vault instance.
+      KES returns an error if the configured |KMS| service is sealed.
+   
+      If you restart or otherwise seal your vault instance, KES cannot perform any cryptographic operations against the vault.
+      You must unseal the Vault to ensure normal operations.
+   
+      See the documentation for your chosen |KMS| solution for more information on whether unsealing may be required.
+
+.. cond:: k8s
+
+   .. include:: /includes/k8s/common-minio-kes.rst
+      :start-after: start-kes-prereq-hashicorp-vault-desc
+      :end-before: end-kes-prereq-hashicorp-vault-desc
+
+Refer to the configuration instruction in the :kes-docs:`KES documentation <>` for your chosen supported |KMS|:
+
+- :kes-docs:`AWS Secrets Manager <integrations/aws-secrets-manager/>`
+- :kes-docs:`Azure KeyVault <integrations/azure-keyvault/>`
+- :kes-docs:`Entrust KeyControl <integrations/entrust-keycontrol/>`
+- :kes-docs:`Fortanix SDKMS <integrations/fortanix-sdkms/>`
+- :kes-docs:`Google Cloud Secret Manager <ntegrations/google-cloud-secret-manager/>`
+- :kes-docs:`Hashicorp Vault <integrations/hashicorp-vault-keystore/>`
+- :kes-docs:`Thales CipherTrust Manager (formerly Gemalto KeySecure) <integrations/thales-ciphertrust/>`
+
+
+.. cond:: linux or macos or windows
+
+   Deploy or Ensure Access to a MinIO Deployment
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   .. include:: /includes/common/common-minio-kes.rst
+      :start-after: start-kes-new-existing-minio-deployment-desc
+      :end-before: end-kes-new-existing-minio-deployment-desc
+
+.. cond:: container
+
+   Install Podman or a Similar Container Management Interface
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   .. include:: /includes/container/common-deploy.rst
+      :start-after: start-common-prereq-container-management-interface
+      :end-before: end-common-prereq-container-management-interface
+
+.. The included file has the correct header structure.
+   There are slight divergences between platforms so this ends up being easier compared to cascading conditionals to handle little nitty-gritty differences.
+
+.. |namespace| replace:: minio-kes-vault
+
+.. cond:: container
```

Comment:
So I think we can remove *all* of these now. They were only necessary when we were documenting the configuration file. Test to confirm.

Reply:
No, we still need them. We document the MinIO configuration file for each platform using platform-specific means.